UI/window-basic-preview.cpp

```diff
@@ -377,6 +378,40 @@ void OBSBasicPreview::GetStretchHandleData(const vec2 &pos)
 	}
 }
 
+void OBSBasicPreview::keyPressEvent(QKeyEvent *event)
+{
+	if (locked || scaled || event->isAutoRepeat()) {
+		OBSQTDisplay::keyPressEvent(event);
+		return;
+	}
+
+	switch (event->key()) {
+		case Qt::Key_Space:
+			setCursor(Qt::OpenHandCursor);
+			scrollMode = true;
+			break;
+	}
+
+	OBSQTDisplay::keyPressEvent(event);
+}
+
+void OBSBasicPreview::keyReleaseEvent(QKeyEvent *event)
+{
+	if (event->isAutoRepeat()) {
+		OBSQTDisplay::keyReleaseEvent(event);
+		return;
+	}
+
+	switch (event->key()) {
+		case Qt::Key_Space:
```

Comment:
Style:  Don't indent case statements in a switch-case.  Align switch and its subordinate case labels.  Indent contents of each case.